client/.env.sample

```
@@ -1,15 +1,12 @@
-APP_URL="http://localhost:9090"
-LOGFIRE_ENABLED="false"
-LOGFIRE_TOKEN=""
-MCP_AUTH_KEY="Authorization"
-MCP_AUTH_VALUE_PREFIX=""
-MCP_URL="https://app.getgram.ai/mcp/your-server
-OAUTH_SERVER_URL="http://localhost:8080"
-OAUTH_USER_AUTH_CLIENT_ID="xqwcYT7SB6lrUIElPyvn7ww8s577UTUCRRmwTTKN"
-OPENAI_API_KEY=""
+CLIENT_URL="htpps://oauth-mcp-client.local"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)